tests/conftest.py

```diff
@@ -255,3 +261,9 @@ def traffic_rule_status_fixture() -> int:
 def wlan_data_fixture() -> list[dict[str, Any]]:
     """WLAN data."""
     return []
+
+
+@pytest.fixture(name="vouchers_payload")
+def vouchers_data_fixture() -> list[dict[str, Any]]:
```

Comment:
```suggestion
def voucher_data_fixture() -> list[dict[str, Any]]:
```
lets do singular on these ones